KlipperPreprocessor.py

```diff
@@ -186,14 +186,16 @@ def return_processed_data(self, filename: str, total_layers: int) -> List[str]:
                     data.append("\n".join(layer))
                     layer = []
                 else:
-                    layer.append(line)
-
                     if add_set_print_stats_info:
                         if line.startswith(';LAYER:'):
                             current_layer += 1
-                            layer.append("SET_PRINT_STATS_INFO CURRENT_LAYER=%s" % (current_layer,))
+                            layer.append("SET_PRINT_STATS_INFO [CURRENT_LAYER=%s]" % (current_layer,))
```

Comment:
The `[` and `]` in Klipper documentation means the enclosed parameter is optional.

One is not supposed to actually use it on the code.